scripts/extras/foglamp.sudoers_rh

```diff
@@ -1 +1 @@
-%sudo ALL=(ALL) NOPASSWD: /usr/bin/yum -y update, /usr/bin/yum -y install foglamp, /usr/bin/yum -y install /usr/local/foglamp/data/plugins/foglamp*.rpm, /usr/bin/yum list
+%sudo ALL=(ALL) NOPASSWD: /usr/bin/yum -y update, /usr/bin/yum -y install foglamp, /usr/bin/yum -y install /usr/local/foglamp/data/plugins/foglamp*.rpm, /usr/bin/yum list, /usr/bin/yum -y install foglamp*
```

Comment:
For this case we need to check and retest `/usr/bin/yum -y install foglamp` **Vs** `/usr/bin/yum -y install foglamp*`